addons/jest/README.md

```diff
@@ -0,0 +1,133 @@
+# Storybook addon Jest
+
+Brings Jest results in storybook.
+
+[![Storybook Jest Addon Demo](@storybook/addon-jest.gif)](https://@storybook/addon-jest-example.herokuapp.com/)
+
+> Checkout the above [Live Storybook](https://@storybook/addon-jest-example.herokuapp.com/).
+
+## Getting started
+
+### Install
+
+`npm install --save-dev @storybook/addon-jest`
+
+or
+
+`yarn add --dev @storybook/addon-jest`
+
+### Jest Configuration
+
+When running **Jest**, be sure to save the results in a json file:
+
+`package.json`
+
+```js
+"scripts": {
+  "test": "jest --json --outputFile=.jest-test-results.json"
+}
+```
+
+Add it the result file to `.gitignore`:
+
+```
+.jest-test-results.json
+```
+
+**Known issue**: if you use a deploy script using for example `gh-pages`, be sure to not put
+the `test` script that write the result in part of the script process (in `predeploy` for example).
+Instead use a different script:
+
+```json
+"scripts": {
+  "test:output": "jest --json --outputFile=.jest-test-results.json",
+  "test": "jest",
+  "prebuild:storybook": "npm run test",
+  "build:storybook": "build-storybook -c .storybook -o build/",
+  "predeploy": "npm run build:storybook",
+  "deploy": "gh-pages -d build/",
+}
+```
+
+Then in dev use:
+
+```sh
+npm run test:output -- --watch
+```
+
+When deploying:
+
+```sh
+npm run deploy
+```
+
+### Register
+
+Register addon at `.storybook/addons.js`
+
+```js
+import '@storybook/addon-jest/register';
+```
+
+## Usage
+
+Assuming that you have created a test files `MyComponent.test.js` and `MyOtherComponent.test.js`
+
+In your `story.js`
+
+```js
+import jestTestResults from '../.jest-test-results.json';
```

Comment:
Does that mean that you have to run the tests before ever starting storybook?

Reply:
Yes, edited the readme to make this clear.

For demo purposes I've added the testresults-file to source control.

Reply:
> For demo purposes

What exactly does it demonstrate? Users are not supposed to create those files manually, are they?